assignments/closure.js

```diff
@@ -1,11 +1,34 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+const decounter = () => {
+  let count1 = 10;
+  return function() {
+    count1 = count1 - 1;
+    return count1;
+  }
+}
 
+const newCounter2 = decounter();
+
+console.log(newCounter2())
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
+  
+  let count = 0;
+  return function() {
+    count = count + 1;
```

Comment:
this should be a call back which means you would want that second function to be outside of the this function. Then you can just do a ++count.